### ISSUE TITLE
Create a flag to untie images from bundle

### ISSUE THREAD

**Issue by freshcot** [werf/werf#6472]
### Before proceeding

- [X] I didn't find a similar [issue](https://github.com/werf/werf/issues)

### Problem

Hello, could you please add flag to `werf bundle publish` and `werf bundle copy` commands which will allow to untie built images from the bundle

### Solution (if you have one)

flag could be `--with-images=true`

### Additional information

Sometimes I need to have bundle without images (only helm chart)

**Comment by QinZhanlong** [comment]
> @freshcot I see that there is a --without-images option in `werf bundle publish` to control whether to build images.

**Comment by alexey-igrychev** [comment]
@freshcot You’re right; the `--without-images` option in `werf bundle publish` allows control over whether images should be built. I see three scenarios related to this issue:

1. `werf bundle publish` (no images)
In this case, the published bundle doesn’t use images, and there’s no need to build the ones defined in werf.yaml. ✅ The --without-images option perfectly suits this scenario. 

2. `werf bundle copy` (optional copying of images)
When copying a bundle from one container registry to another, copying images might not be necessary. This is useful when the images are already present in the target registry.

3. `werf bundle publish` (disabling copying of images)
This is similar to the werf bundle copy, but the behavior (potentially optional, and you can force it when using `werf bundle copy`) is embedded into the bundle itself. This can be useful in cases where images are stored separately from the bundle, for example, in an independent registry.

**Comment by alexey-igrychev** [comment]
@freshcot Hey there! Could you share a bit more about your case?

**Comment by freshcot** [comment]
Hi @alexey-igrychev , so my case is when I using `werf bundle` to publish images and chart, I want to download chart with `werf bundle copy` and replace some values so I will not be obligated to use `--set` while I will be deploying this chart with `helm`
and during this operation (copy) werf downloading images as well

**Comment by alexey-igrychev** [comment]
Thus, the second and third cases I wrote about in the previous message work well for you. 

@QinZhanlong, if you're eager to put this into action, I’m here to assist you 🙌🏼

**Comment by alexey-igrychev** [comment]
Status check: only the first of the three scenarios above is implemented — `werf bundle publish --without-images`. `werf bundle copy` still has no way to skip copying images, and a bundle cannot record that its images live in the original registry. Keeping this open for those two.
